src/scenes/Partners/Detail/AddressCard.tsx

```diff
@@ -0,0 +1,58 @@
+import {
```

Comment:
Let me know if you think I should move this and `PartnerTypesCard.tsx` to the components directory, wasn't sure if this may be reusued

Reply:
I don't see anything else in the existing designs that resembles this, so I say we just leave it here unless it changes to more closely resemble another existing component. I suppose there's a chance we'll need it or something very similar to it again elsewhere in the app, won't see it in here, and will end up duplicating some code. But ideally any time we need to create some UI we should be looking in Figma to see if it already exist in the designs for a scene we've already implemented, and if so, we ought to find it then.